### DISCUSSION TITLE
Slovak Translation

### DISCUSSION THREAD

**Discussion by m3nu** [borgbase/vorta#978] [Translations]
> @jose1711: Hi, please add me to the thread. I'd like to work on translation to Slovak. Thanks!

Thanks for adding Slovak. Much appreciated. 👍

To continue, please see our language [contribution site](https://vorta.borgbase.com/contributing/translations/). Since we use Transifex for translations, please sign up there and [suggest](https://www.transifex.com/borgbase/vorta/dashboard/) Slovak. I'll add you as translator there and once the translation is complete enough, it will be added to Vorta automatically.

**Comment by jose1711** [comment]
Thank you, Slovak requested via Transifex.

**Reply by m3nu** [reply]
Thanks. Added Slovak as language and yourself `jose1711` as translator.
